packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -438,6 +421,23 @@ class _ModalBottomSheetState<T> extends State<_ModalBottomSheet<T>> {
           ),
         );
       },
+      child: BottomSheet(
```

Comment:
This is another example where the lint is overly strict as the `builder` and the `child` both generate children rather than properties so I don't care about the order from the perspective of UI guides.